CHANGELOG.md

```diff
@@ -5,8 +5,10 @@
 ## Added
 
 ## Changed
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.152.1`
 
 ## Fixed
+- downgrade rust in core to avoid wrong Windows malware detection https://github.com/deltachat/deltachat-core-rust/issues/6338
```

Comment:
Looking at previous entries, we seem to usually bunch together changelog entries for Core upgrades

Reply:
Well between 1.152.0 and 1.152.1 there is nothing else than what is in this line